contracts/Rewards.sol

```diff
@@ -376,70 +160,11 @@ contract Rewards is IRewards, ContractRegistryAccessor, ERC20AccessorWithTokenGr
 
         uint256 remainder = amount.sub(guardianFee.mul(n));
         if (remainder > 0) {
-            fillFeeBucket(_bucketTime(now), remainder, isCertified);
+            // TODO probably an overkill...
```

Comment:
An overkill and confusing in terms of events, how about storing the reminder locally and add to the next collectFees?

Reply:
Sounds good, althouhg if we move the wallets to an `approve` based interface it would complicate things (funds should be transffered from both the fees wallet and rewards).
Another option - keep track on the sum of remainders, and add an external function that transffers it to the fees wallet like done her.
Another option - ignore the remainder and let it rot in the rewards contract..

Reply:
May ignore it and leave in the rewards contract.